test/test-integration/src/test/java/org/wso2am/micro/gw/tests/common/model/ApplicationDTO.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2am.micro.gw.tests.common.model;
+
+import java.io.Serializable;
+
+/**
+ * Application model
+ */
+public class ApplicationDTO implements Serializable {
+
+    private int id;
+    private String name;
+    private String tier;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getTier() {
+        return tier;
+    }
+
+    public void setTier(String tier) {
+        this.tier = tier;
+    }
+}
```

Comment:
Shall we add a new line at the EOF?

Reply:
And please check the other files in this PR and do the same.

Reply:
Fixed in 3b6cc6755810d748cd3de4fb25696a5692c08e78